### PR TITLE
LogMeIn.GoToMeeting version 10.15.19228

### PR DESCRIPTION
**Pull request by Amoenus**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15619)

### CODE REVIEW

File: manifests/l/LogMeIn/GoToMeeting/10.15.19228/LogMeIn.GoToMeeting.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: LogMeIn.GoToMeeting
+PackageVersion: 10.15.19228
```
Comment (jedieaston) [inline manifests/l/LogMeIn/GoToMeeting/10.15.19228/LogMeIn.GoToMeeting.installer.yaml:4]:
```suggestion
PackageVersion: 10.15.0.19228

```

File: manifests/l/LogMeIn/GoToMeeting/10.15.19228/LogMeIn.GoToMeeting.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: LogMeIn.GoToMeeting
+PackageVersion: 10.15.19228
+PackageLocale: en-US
+Publisher: LogMeIn
+PackageName: GoToMeeting
```
Comment (jedieaston) [inline manifests/l/LogMeIn/GoToMeeting/10.15.19228/LogMeIn.GoToMeeting.locale.en-US.yaml:7]:
```suggestion
PackageName: GoToMeeting 10.15.0.19228

```

File: manifests/l/LogMeIn/GoToMeeting/10.15.19228/LogMeIn.GoToMeeting.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: LogMeIn.GoToMeeting
+PackageVersion: 10.15.19228
```
Comment (jedieaston) [inline manifests/l/LogMeIn/GoToMeeting/10.15.19228/LogMeIn.GoToMeeting.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 10.15.0.19228

```

File: manifests/l/LogMeIn/GoToMeeting/10.15.19228/LogMeIn.GoToMeeting.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: LogMeIn.GoToMeeting
+PackageVersion: 10.15.19228
```
Comment (jedieaston) [inline manifests/l/LogMeIn/GoToMeeting/10.15.19228/LogMeIn.GoToMeeting.yaml:4]:
```suggestion
PackageVersion: 10.15.0.19228

```

**Review by jedieaston — CHANGES_REQUESTED** [review]
Once you change these and move the manifest to the folder manifests/l/LogMeIn/GoToMeeting/10.15.0.19228/, you should be good to go.

**Comment by jedieaston** [comment]
The log entry:

```


 Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\l\LogMeIn\GoToMeeting\10.15.19228', expected 'manifests\l\LogMeIn\GoToMeeting\10.15.0.19228'

```

**Comment by Amoenus** [comment]
Made the change. Sorry for the delay, was looking for my PGP token passphrase

File: manifests/l/LogMeIn/GoToMeeting/10.15.0.19228/LogMeIn.GoToMeeting.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate
+
+PackageIdentifier: LogMeIn.GoToMeeting
+PackageVersion: 10.15.0.19228
+
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
```
Comment (ItzLevvie) [inline manifests/l/LogMeIn/GoToMeeting/10.15.0.19228/LogMeIn.GoToMeeting.installer.yaml:8]:
```suggestion
  Architecture: x86
```

File: manifests/l/LogMeIn/GoToMeeting/10.15.0.19228/LogMeIn.GoToMeeting.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: LogMeIn.GoToMeeting
+PackageVersion: 10.15.0.19228
+
+PackageLocale: en-US
+Publisher: LogMeIn
```
Comment (ItzLevvie) [inline manifests/l/LogMeIn/GoToMeeting/10.15.0.19228/LogMeIn.GoToMeeting.locale.en-US.yaml:7]:
```suggestion
Publisher: LogMeIn, Inc.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]
image

**Pull request merged**